mesh/mesh.go

```diff
@@ -25,6 +28,9 @@ const (
 
 var TRUE = []byte{1}
 var FALSE = []byte{0}
+var LATEST = []byte("latest")
+var VALIDATED = []byte("validated")
+var TORTOISE = []byte("tortoise")
```

Comment:
can't these be consts ?

Reply:
not if i want them to be of type []byte, maybe i can use strings but that  would incur casting and i prefer this